src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -73,21 +76,25 @@ public void process(Objection objection, String httpRequestId)
 
         validateObjectionStatus(objection, httpRequestId);
 
-        // TODO update status to PROCESSING
+        out.setStatus(ObjectionStatus.PROCESSING);
+        out.setCreatedOn(LocalDateTime.now());
```

Comment:
If the idea is to update the status and timestamp as the code progresses, don't you have to save the entity to Mongo at each of these steps?